ulab_Crunch_Numbers_Fast/benchmark.py

```diff
@@ -1,11 +1,21 @@
-import time
+import utime
```

Comment:
This is not correct for CircuitPython.  Please check that the code runs on CircuitPython, as that's what the Guide covers.

Reply:
This was sort of stupid, because `micropython` would have worked with `time`, too. I will fix the code.